README.md

```diff
@@ -99,3 +138,7 @@ console.log(i)
 
 -- YOUR ANSWER HERE --
 
+Bob
+curly
+1
+error
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.